tests/logictest/suites/gen/20+_others/20_0001_planner_v2

```diff
@@ -1259,56 +1259,6 @@ select (select number from numbers(10) as t1 where t.number = t1.number) from nu
 8
 9
 
-skipif clickhouse
```

Comment:
Removed because there is a specific dir for explain in logic test https://github.com/datafuselabs/databend/blob/main/tests/logictest/suites/gen/04_explain/04_0002_explain_v2 cc @ZeaLoVe